webapp/src/components/systemConsole/teamFilter/teamIcon/teamIcon.tsx

```diff
@@ -15,16 +15,9 @@ export type Props = {
     data: DropdownOption;
 }
 function TeamIcon({data}: Props) {
-    // @ts-expect-error last_team_icon_update
-    // field will be included in latest types release,
-    // but its there in the data already
-    if (data.raw.last_team_icon_update) {
-        // last_team_icon_update is set whenever a team icon is set or updated.
-        // So, a truthy value of `last_team_icon_update` field indicates precense of
-        // team icons, so here we use it to show the team icon image.
-
-        // @ts-expect-error sane reason as above about the last_team_icon_update field
-        const teamURL = Client4.getTeamIconUrl(data.value, (data.raw as Team)?.last_team_icon_update);
+    const teamIconLastUpdated = (data.raw as Team).last_team_icon_update;
```

Comment:
We can leave it like this (it is good enough and we control the data, so probably this is way more readable that what I am going to propose), but type casting is quite insecure in Typescript (unknown can be cast to anything) and we should try to avoid it.

In order to make sure it is a team, the "Typescript way" is to use a guard. Another way to handle this, is to check if `last_team_icon_update` exist in raw.

Nevertheless, as I said, let's keep it like this, since it is more readable and has less boilerplate.